spec/requests/show_workflow_template_spec.rb

```diff
@@ -26,10 +26,9 @@
   end
 
   context 'for an nonexistent template' do
-    it 'returns 404' do
+    it 'returns 400' do
```

Comment:
This changed to 400 because we're now validating the workflow names per a new enum, right?

Reply:
Correct.